src/makefile

```diff
@@ -65,15 +65,12 @@ $(PREFIX)/bin/psql: postgresql-$(POSTGRES_VERSION)/GNUmakefile
 # that postgres will be compiled and linked against our own versions
 # of libraries like openssl, instead of system provided versions
 postgresql-$(POSTGRES_VERSION)/GNUmakefile: $(PREFIX)/lib/libssl.dylib $(PREFIX)/lib/libxml2.dylib $(PREFIX)/lib/libedit.dylib ${PREFIX}/include/uuid.h postgresql-$(POSTGRES_VERSION)/configure
-	cd "postgresql-$(POSTGRES_VERSION)" && export PATH="$(PREFIX)/bin:$$PATH" && ./configure --prefix="$(PREFIX)" --with-includes="$(PREFIX)/include" --with-libraries="$(PREFIX)/lib" --enable-thread-safety --with-openssl --with-gssapi --with-bonjour --with-krb5 --with-libxml --with-libxslt --with-perl --with-python --with-libedit-preferred --with-ossp-uuid
+	cd "postgresql-$(POSTGRES_VERSION)" && export PATH="$(PREFIX)/bin:$$PATH" && ./configure --prefix="$(PREFIX)" --with-includes="$(PREFIX)/include" --with-libraries="$(PREFIX)/lib" --enable-thread-safety --with-openssl --with-gssapi --with-bonjour --with-krb5 --with-libxml --with-libxslt --with-perl --with-python --with-libedit-preferred --with-uuid=e2fs
 
 # after extracting PostgreSQL sources, we must patch uuid-ossp so it builds on OSX
+#ossp no longer builds with 9.4.x+, use e2fs-libuuid 
 postgresql-$(POSTGRES_VERSION)/configure: postgresql-$(POSTGRES_VERSION).tar.bz2
 	$(TAR) "postgresql-$(POSTGRES_VERSION).tar.bz2"
-	echo '#define _XOPEN_SOURCE' >"postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched"
-	cat "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c" >>"postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched"
-	mv "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched" "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c"
-	touch $@
```

Comment:
I see, I had thought it was for the patched ossp file, I'll put it back in
when I rebuild

On Mon, Nov 30, 2015 at 10:35 AM, Larry Shaffer <notifications@github.com>
wrote:

> In src/makefile
> <https://github.com/boundlessgeo/PostGISApp/pull/6#discussion_r46182834>:
>
> >  postgresql-$(POSTGRES_VERSION)/configure: postgresql-$(POSTGRES_VERSION).tar.bz2
> >  	$(TAR) "postgresql-$(POSTGRES_VERSION).tar.bz2"
> > -	echo '#define _XOPEN_SOURCE' >"postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched"
> > -	cat "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c" >>"postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched"
> > -	mv "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c.patched" "postgresql-$(POSTGRES_VERSION)/contrib/uuid-ossp/uuid-ossp.c"
> > -	touch $@
>
> @mtCarto <https://github.com/mtCarto>, the touch $@ command should not be
> dropped, as it helps keep dependency rebuilding in check. The command will
> ensure configure is newer and trigger a re-configure of the project,
> *each* time the archive is re-expanded.
>
> See:
> http://kludger.blogspot.com/2013/06/simple-makefile-dependencies-using.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/boundlessgeo/PostGISApp/pull/6/files#r46182834>.
>



-- 
Morgan Thompson
Junior Engineer | Boundless
mthompson@boundlessgeo.com